itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -107,8 +107,8 @@ public void testCreateFunctionIncrementalReplication() throws Throwable {
   @Test
   public void testDropFunctionIncrementalReplication() throws Throwable {
     primary.run("CREATE FUNCTION " + primaryDbName
-        + ".testFunction as 'com.yahoo.sketches.hive.theta.DataToSketchUDAF' "
-        + "using jar  'ivy://com.yahoo.datasketches:sketches-hive:0.8.2'");
+        + ".testFunction as 'hivemall.tools.string.StopwordUDF' "
+        + "using jar  'ivy://io.github.myui:hivemall:0.4.0-2'");
     WarehouseInstance.Tuple bootStrapDump = primary.dump(primaryDbName, null);
     replica.load(replicatedDbName, bootStrapDump.dumpLocation)
         .run("REPL STATUS " + replicatedDbName)
```

Comment:
My query was, if CREATE FUNCTION fails due to jar unaccessible or any other reasons, then bootstrap won't have it and so lastReplId verification alone won't be enough to ensure that. Apparently, another test testCreateFunctionIncrementalReplication uses the same jar and verifies it. That's fine.